Chinese/Lady/Lady.md

```diff
@@ -10,5 +10,5 @@
 凤姐的气场是众所周知的强悍。
 而八戒的审美又是得天独厚的独特。
 这一章正是:
-凤姐初来奉国寺
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.